### PR TITLE
replace RKVST favicon with DataTrails

### PR DESCRIPTION
**Pull request by pcnorth**
Replace the RKVST tick with the DataTrails trail icon.

### CODE REVIEW

File: layouts/partials/head/favicons.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <meta name="theme-color" content="{{ $.Site.Params.themeColor }}">
-<link rel="apple-touch-icon" sizes="180x180" href="{{ "apple-touch-icon.png" | relURL }}">
-<link rel="icon" type="image/png" sizes="32x32" href="{{ "favicon-32x32.png" | relURL }}">
-<link rel="icon" type="image/png" sizes="16x16" href="{{ "favicon-16x16.png" | relURL }}">
+<!--link rel="apple-touch-icon" sizes="180x180" href="{{ "apple-touch-icon.png" | relURL }}"-->
+<link rel="icon" type="image/png" sizes="49x49" href="{{ "DataTrails_Favicon.png" | relURL }}">
```
Comment (JAG-UK) [inline layouts/partials/head/favicons.html:3]: This is really supposed to be 49, not 48?

File: layouts/partials/head/favicons.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <meta name="theme-color" content="{{ $.Site.Params.themeColor }}">
-<link rel="apple-touch-icon" sizes="180x180" href="{{ "apple-touch-icon.png" | relURL }}">
-<link rel="icon" type="image/png" sizes="32x32" href="{{ "favicon-32x32.png" | relURL }}">
-<link rel="icon" type="image/png" sizes="16x16" href="{{ "favicon-16x16.png" | relURL }}">
+<!--link rel="apple-touch-icon" sizes="180x180" href="{{ "apple-touch-icon.png" | relURL }}"-->
```
Comment (JAG-UK) [inline layouts/partials/head/favicons.html:2]: If this isn't used then delete it, don't just comment it out.

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: layouts/partials/head/favicons.html (hunk 1 above)
Comment (pcnorth) [reply layouts/partials/head/favicons.html:3]: It was the size that I downloaded from sharepoint. I'll change it to 48

File: layouts/partials/head/favicons.html (hunk 2 above)
Comment (pcnorth) [reply layouts/partials/head/favicons.html]: Done

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**